ui/app/common/displaycontrols/navigationlinks/directives/navigationLinks.js

```diff
@@ -67,6 +67,9 @@ angular.module('bahmni.common.displaycontrol.navigationlinks')
             $scope.getUrl = function (link) {
                 var url = getFormattedURL(link);
                 window.open(url, link.title);
+                if (url.includes("/bahmni_config/esanjeevani")) {
+                    logAuditForESanjeevaniVisit();
```

Comment:
This code needs to be outside of bahmniapps codebase (else Bahmni will get strongly coupled with eSanjeevani). We don't want eSanjeevani mention within Bahmni core code.

Reply:
Got it @gsluthra, the logic for creating log for E-Sanjeevani visit has now been shifted to the [index.html](https://github.com/Bahmni-HWC/clinic-config/blob/main/esanjeevani/index.html) page. When the user clicks on the `Login` button, the visit will be logged against our audit log. 

Please refer : https://github.com/Bahmni-HWC/clinic-config/pull/59.